packages/ruby/spec/readme/har/response_serializer_spec.rb

```diff
@@ -103,6 +103,35 @@
       expect(json.dig(:content, :size)).to eq 0
       expect(json.dig(:content, :mimeType)).to eq ""
     end
+
+    it "handles multiple JSON mime types" do
+      json_mime_types = [
+        "application/json",
+        "application/x-json",
+        "text/json",
+        "text/x-json",
+        "+json"
+      ]
+      request = build_request
+      body = [{result: "value"}.to_json]
+
+      json_mime_types.each do |mime_type|
+        response = build_response(
+          status: 200,
+          content_type: mime_type,
+          body: body
+        )
+        serializer = Readme::Har::ResponseSerializer.new(
+          request,
+          response,
+          Filter.for
+        )
+        json = serializer.as_json
+
+        expect(json[:content][:text]).to eq body.first
+        expect(json[:content][:mimeType]).to eq mime_type
```

Comment:
Was this test red before you implemented the change? My guess is that this would be true no matter the mime type as long as there is some kind of body

Reply:
This test was red. The JSON content type was hardcoded as "application/json"

Reply:
Oh I see, so it failed for `application/json` but passed for the others.